doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -1045,3 +1046,16 @@
     iox::byte m_size;
     ```
 
+48. Move conversion methods from `duration.hpp` to `iceoryx_dust`
+
+    ```cpp
+    //before
+    std::chrono::milliseconds chronoDuration = 1_ms;
+    iox::units::Duration ioxDuration(chronoDuration);
+
+    //after
+    #include "iceoryx_dust/cxx/std_chrono_support.hpp"
+
+    std::chrono::milliseconds chronoDuration = 1_ms;
+    iox::units::Duration ioxDuration = into<iox::units::Duration>(chronoDuration);
```

Comment:
With this code one would introduce Axivian warnings ;)

If you need another commit please consider changing this either to `auto ioxDuration = ...` or ``iox::units::Duration ioxDuration {...};